src/client.c

```diff
@@ -952,6 +952,7 @@ static bool handle_client_work(PgSocket *client, PktHdr *pkt)
 	/* tag the server as dirty */
 	client->link->ready = false;
 	client->link->idle_tx = false;
+	client->link->generation++; /* TODO figure out what happens on overflow */
```

Comment:
I think it can reasonably be assumed that this never overflows, since it's a 64bit uint. Which means that even when incrementing this every nanosecond this would take 584 years to overflow. Any restart within these 584 years would reset this number to 0. 

Source: https://github.com/jbenet/nanotime#range